hamocc/sedshi.F90

```diff
@@ -219,7 +219,7 @@ SUBROUTINE SEDSHI(kpie,kpje,omask)
 
 ! fill the last active layer
         refill=seddef/(buried+1.e-10) 
-        frac = porsol(ks)*seddw(ks) !changed k to ks, ik
+        frac = porsol(i,j,ks)*seddw(ks) !changed k to ks, ik
```

Comment:
```suggestion
        frac = porsol(i,j,ks)*seddw(ks)
```
another comment that doesn't seem to add much info

Reply:
I can delete it, sure.